_layouts/about.html

```diff
@@ -9,7 +9,7 @@
            {% if site.title == "blank" -%}<span class="font-weight-bold">{{ site.first_name }}</span> {{ site.middle_name }} {{ site.last_name }}{%- else -%}{{ site.title }}{%- endif %}
           </h1> -->
           <!-- add image for heading -->
-          <img src="{{ site.baseurl }}/assets/img/EKLAVYA_2024__6_-removebg-preview.png" alt="heading" style="width: 500px; height: 100px; object-fit: cover; object-position: 50% 25%;">
+          <img src="{{ site.baseurl }}/assets/img/EKLAVYA_2024__6_-removebg-preview.png" alt="heading" style="width: 500px; height: 100px; object-fit: fill; object-position: 50% 25%;">
```

Comment:
If you won't specify height, it will maintain the ratio only

Reply:
![image](https://github.com/SRA-VJTI/eklavya.github.io/assets/72208314/0a6a9da8-f733-4ad3-846e-81d60071b509)
Is this fine, I have set it to auto

Reply:
yeah looks gr8